operator/provider_aws_flags.go

```diff
@@ -45,6 +45,9 @@ func init() {
 
 	flags.Bool(operatorOption.UpdateEC2AdapterLimitViaAPI, false, "Use the EC2 API to update the instance type to adapter limits")
 	option.BindEnv(operatorOption.UpdateEC2AdapterLimitViaAPI)
+	flags.MarkDeprecated(operatorOption.UpdateEC2AdapterLimitViaAPI, "This option has been deprecated and will be removed in v2.0")
```

Comment:
It's very unlikely that we will release 2.0 next ;-) Let's assume it will be 1.10.

Reply:
Ok, I will change it to 1.10.

Reply:
Done